docs/release_notes.rst

```diff
@@ -31942,9 +31937,9 @@ backward compatibility issues and work with the default 10 job
 limit in single calls to the IBM Q API v2.
 
 
-***********
+##########*
```

Comment:
There is a stray `*` here (along with on the underline as well. Also on 0.10 too